package.json

```diff
@@ -41,6 +41,10 @@
     "boring-avatars": "^1.7.0",
     "d3-axis": "^3.0.0",
     "d3-scale": "^4.0.2",
+    "d3-scale-chromatic": "3.1.0",
+    "d3-time-format":"4.1.0",
+    "d3-time":"3.1.0",
+    "d3":"7.9.0",
```

Comment:
Having d3 fully means larger bundle size , all the d3 functionalities are modularized well into smaller packages. Instead of having full d3 package, can you try to use modularized packages ?